Documentation/D2.md

```diff
@@ -0,0 +1,43 @@
+# Programming Hat (D2 Progress Report)
+
+## Team Members:
+* Austin Soriano
+* *Casey Key*
+* Zach Garno
+
+## Project Status
+This project is currently active. All hardware has been purchased and the Raspberry Pi Zero W (RPi) to motor switch has been assembled.
+### Impediments:
+During our first meeting we discovered that:
+ 1. The only interface to the RPi was a micro usb. We rewired a usb keyboard with a micro usb to solve this problem. As this is only necessary for manufacturing this will not be included in the cost of the final produce.
+ 2. The switch purchased required a diod which delayed initial testing. This part is now included in the modified parts list below.
+
+## Metrics:
+We plan on evaluating the following metrics in order:
+* Response Time ( Time between local commit, remote push, or pull request and recorded output event from the RPi code)
+* Cost and marketability ( A servey with standard questions that contribute to the goal of obtaining the level of interest, user acceptance, and device pricing)
+* Power consumption (Using standard RPi power statistics to estimate power consumption)
+* Security evaluation (Evaluate any security concerns using published documents for existing methods and hardware. Running a network sniffing tool (wire shark) to determine what, if any information is leaked through network traffic)
+
+## Hardware Proof:
+![Image of Hardware Parts](https://github.com/zach-garno/CS-370-Pi-Project/images/Parts.jpg)
+
+## Parts List
+* Raspberry Pi Zero W (Ordered from Sparkfun) $10 
+* 12 RGB LED’s (Ordered from Sparkfun) $6
+* Power bank
```

Comment:
We are now using a 20700 Li-ion battery with a battery charging/boost board and battery sled.

Reply:
Done. What is / was the cost? We will eventually need a full price list for the marketability evaluation.